webapp/turntable/models.py

```diff
@@ -46,10 +46,17 @@ class Producer(db.Model):
     description = db.Column(db.String(), nullable=True)
     ptype = db.Column(db.String(), nullable=False)
 
+    def is_github(self):
+        return self.ptype == 'github'
+    
     @property
     def url(self):
         return 'apipivots.io/pi/{}'.format(self.url_path)
 
+    @property
+    def github_secret(self):
+        return 'my-super-secret'
```

Comment:
shouldn't you return a config.secret or class variable instead of const string ?

Reply:
:+1: WIP